test/services/EventReportServiceSpec.scala

```diff
@@ -209,31 +209,31 @@ class EventReportServiceSpec extends AsyncWordSpec with Matchers with MockitoSug
 
 
   "getEvent" must {
```

Comment:
Have we replaced the event1 tests completely with event22? Are we losing coverage?

Reply:
These tests weren't to test event1. At the time those tests were implemented, it was merely to check that data was being cached regardless of what the data was. Have now updated them so that they're relevant to the event caching for event 22 and to check that they are transforming the payload correctly